src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/GeneralPartnerControllerValidationTest.java

```diff
@@ -177,4 +189,71 @@ void shouldReturn400WhenCreatingGeneralPartnerLegalEntityWithWrongCountry() thro
                 .andExpect(status().isBadRequest())
                 .andExpect(jsonPath("$.['errors'].['data.legalEntityRegistrationLocation']").value("Legal entity registration location must be valid"));
     }
+
+    @Nested
+    class ValidatePartnership {
+        @Test
+        void shouldReturn200IfNoErrors() throws Exception {
+            when(generalPartnerService.validateGeneralPartner(transaction, GENERAL_PARTNER_ID))
+                    .thenReturn(new ArrayList<>());
```

Comment:
Don't think this needs mocking if there are no errors.

Reply:
You're right. Will remove this.

Reply:
Done.